doc/source/whats_new.rst

```diff
@@ -9,6 +9,10 @@ Current
 Changelog
 ~~~~~~~~~
 
+   - Add support for single dipole fitting by `Eric Larson`_
+
+   - Add support for spherical BEM models by `Eric Larson`_
```

Comment:
... and the fact that you know the surfaces are spherical simplifies some of the math. But I think it still qualifies as a BEM, unless I have my definitions mixed up